src/main/java/io/orangebeard/listener/OrangebeardTestSystemListener.java

```diff
@@ -219,7 +219,7 @@ public void testStarted(TestPage testPage) {
             if (!runContext.suiteExists(suitePath)) {
                 StartSuite startSuite = new StartSuite(testRunUUID, parentSuiteId, description, attributes, List.of(suites));
                 orangebeardClient.startSuite(startSuite);
-                runContext.addSuite(suitePath, UUID.randomUUID());
+                runContext.addSuite(suitePath, UUID.randomUUID(), parentSuiteId, suites);
```

Comment:
Shouldn't we use the actual uuid's from the startSuite call's response here?

Reply:
Hmm maybe, you think that matters? It makes sense of course just I didn't change the implementation because I've never seen a scenario where a map entry is retrieved from run context by providing the suite id it's quite the opposite used there. Plus when the suite is started it returns back a list of suites with a list of uuid's. What do you think?